pkg/cli/update.go

```diff
@@ -0,0 +1,151 @@
+package cli
+
+import (
+	"archive/zip"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"syscall"
+
+	"github.com/google/go-github/github"
+	"github.com/sosedoff/pgweb/pkg/command"
+)
+
+var (
+	latestVersion string
+	workDir       = "/tmp"
+	distName      = fmt.Sprintf("pgweb_%s_%s", runtime.GOOS, runtime.GOARCH)
+	zipName       = distName + ".zip"
+)
+
+func checkUpdate() {
+
+	if options.DisableCheckUpdate {
+		return
+	}
+
+	client := github.NewClient(nil)
+
+	latestRelease, _, err := client.Repositories.GetLatestRelease(context.Background(), "sosedoff", "pgweb")
```

Comment:
We don't need to pull in the entire github client for this single call. Their API is quite friendly:

```sh
$ curl -s 'https://api.github.com/repos/sosedoff/pgweb/releases/latest' | jq -r '.name'
0.9.11
```

```go
var latestRelease struct{ Name string }

response, err := http.Get("https://api.github.com/repos/sosedoff/pgweb/releases/latest")
if err != nil {
  // TODO
}
defer response.Body.Close()

err = json.NewDecoder(response.Body).Decode(&latestRelease)
if err != nil {
  // TODO
}
```

Reply:
I didn't know about `json.Decode`. Thx

Reply:
Fixed